### DISCUSSION TITLE
Set migrations type postgresql

### DISCUSSION THREAD

**Discussion by Arnaud-Lyard** [mikro-orm/mikro-orm#4822] [General]
**Describe the bug**
This is first time I'm working with MikroORM. I'm trying to enable migrations to go to production with MikroORM. I use a container docker to host a postgresql database but when I run ```npx mikro-orm migration:create --initial``` 

I got the fallowing error :
```Error: No platform type specified, please fill in `type` or provide custom driver class in `driver` option. Available platforms types: [
  'mongo',
  'mysql',
  'mariadb',
  'postgresql',
  'sqlite',
  'better-sqlite'
]```

This is my custom ```database.ts``` file : 

``` 
export const ormConfig = {
  migrations: {
    path: "./src/migrations",
    tableName: "migrations",
    transactional: true,
  },
  user: envConfig.DB_USER,
  password: envConfig.DB_PASSWORD,
  dbName: envConfig.DB_NAME,
  host: envConfig.DB_HOST,
  port: envConfig.DB_PORT,
  type: "postgresql",
  entities: [User, Article, Profile, Player, Match, PlayerToMatch],
  driver: PostgreSqlDriver,
} as Options<PostgreSqlDriver>; 
```
And the config in ```package.json``` : 
```
  "mikro-orm": {
    "useTsNode": true,
    "configPaths": [
      "./src/database.ts",
      "./dist/database.js"
    ]
  },
```

The backend stack is on NodeJS, typescript (v5.0.2), ts-node(v10.9.1), mikro-orm(v5.8.2).

**Comment by B4nan** [comment]
> export const ormConfig = {

You are supposed to be default exporting the config, it's not read at all, and that is your problem. I guess it would make sense to throw in case the config path exists but does not export anything properly, will try to do that in v6.

**Reply by Arnaud-Lyard** [reply]
Thanks for reply, it works like a charm. I have some migration memory on Symfony and the possibility to launch them at start server is so good. May you could force the export default if you have often the question.
